src/views/ipv/pyi-kbv-thin-file.njk

```diff
@@ -0,0 +1,16 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% set pageTitleName = 'pages.pyiKbvThinFile.title' | translate %}
+{% set googleTagManagerPageId = "pyiKbvThinFile" %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">{{'pages.pyiKbvThinFile.header' | translate }}</h1>
+  <p class="govuk-body">{{'pages.pyiKbvThinFile.content.paragraph1' | translate }}</p>
+  <p class="govuk-body">{{'pages.pyiKbvThinFile.content.paragraph2' | translate }}</p>
+  <h2 class="govuk-heading-m">{{'pages.pyiKbvThinFile.content.subHeading' | translate }}</h2>
+  <p class="govuk-body">{{'pages.pyiKbvThinFile.content.paragraph3' | translate }}</p>
+
+  {% include 'shared/journey-next-form.njk' %}
+
+  <p class="govuk-body"><a target="_blank" rel="noopener noreferrer" href="{{'pages.pyiKbvThinFile.content.contactLinkHref' | translate }}" class="govuk-link">{{'pages.pyiKbvThinFile.content.contactLinkText' | translate }}</a></p>
```

Comment:
we can make the contact link universal under a single translation field, i believe its a repeated translation for each page